utils/helper.sh

```diff
@@ -59,6 +59,22 @@ function check_ccloud_v2() {
   return 0
 }
 
+function version_gt() { test "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1"; }
+
+function get_ccloud_version() {
+  ccloud version | grep "^Version:" | cut -d':' -f2 | cut -d'v' -f2
+}
+
+function check_ccloud_version() {
```

Comment:
As a new function, it lost the check that `ccloud` exists in the first place.

If it doesn't exist, user gets error:


```
../../utils/helper.sh: line 65: ccloud: command not found

# Login
couldn't execute "ccloud": no such file or directory
    while executing
"spawn ccloud login --url https://stag.cpdev.cloud"
spawn ccloud login --url https://stag.cpdev.cloud
Failed to log into your cluster.  Please check all parameters and run again
```

Reply:
How do you want me to check?

Reusing this function:
https://github.com/confluentinc/examples/blob/5.3.1-post/utils/helper.sh#L28
Seems improper to me as that config file isn't required to operate ccloud anymore

Reply:
Does this work? https://github.com/confluentinc/examples/blob/5.3.1-post/utils/helper.sh#L23-L26

Reply:
I added a function to check for the binary only and then referenced that from the existing check to not break existing function.  I then reference that in the version check.  Please see if that works.  I tested by removing the binary